examples/ssr-caching/server.js

```diff
@@ -1,6 +1,8 @@
 const express = require('express')
 const next = require('next')
 const LRUCache = require('lru-cache')
+const { BLOCKED_PAGES } = require('next/constants')
+const { isInternalUrl } = require('next/dist/server/utils')
```

Comment:
This is a private API, is seems like it's not needed as `renderAndCache` is a method and shouldn't be called on the javascript files etc